### PR TITLE
Mission feasibility checks: improve correctness of docs, escpecially around MIS_DIST_1WP

### PR DESCRIPTION
**Pull request by sfuhrer**
The docs around mission feasibility are a bit outdated, especially since https://github.com/PX4/PX4-Autopilot/pull/23568.

Related PR in PX4 for param description: https://github.com/PX4/PX4-Autopilot/pull/23873

### CODE REVIEW

File: en/flight_modes_fw/mission.md [hunk 1]
```diff
@@ -76,18 +76,20 @@ For more information about mission planning, see:
...
 
-- First mission item too far away from vehicle ([MIS_DIST_1WP](#MIS_DIST_1WP))
 - Any mission item conflicts with a plan or safety geofence
 - More than one land start mission item defined ([MAV_CMD_DO_LAND_START](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_LAND_START))
 - A fixed-wing landing has an infeasible slope angle ([FW_LND_ANG](#FW_LND_ANG))
 - Land start item (`MAV_CMD_DO_LAND_START`) appears in mission before an RTL item ([MAV_CMD_NAV_RETURN_TO_LAUNCH](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_RETURN_TO_LAUNCH))
 - Missing takeoff and/or land item when these are configured as a requirement ([MIS_TKO_LAND_REQ](#MIS_TKO_LAND_REQ))
 
+Additionally there is a check if the first waypoint is too far from the Home position ([MIS_DIST_1WP](#MIS_DIST_1WP)).
+The user is notified should the check fail, but it has no effect on the validity of a mission plan, meaning that the mission can still be started even if the distance is too high.
```
Comment (hamishwillee) [inline en/flight_modes_fw/mission.md:91]:
```suggestion
The user is notified should the check fail, but it has no effect on the validity of a mission plan (the mission can still be started even if the distance is too great).
```

File: en/flight_modes_fw/mission.md [hunk 2]
```diff
@@ -76,18 +76,20 @@ For more information about mission planning, see:
 
 ## Mission Feasibility Checks
 
-PX4 runs some basic sanity checks to determine if a mission is feasible when it is uploaded, and when the vehicle is first armed.
+PX4 runs some basic sanity checks to determine if a mission is feasible when it is uploaded and before executing a mission.
```
Comment (hamishwillee) [inline en/flight_modes_fw/mission.md:79]: FMI Does this prevent switching to mission mode or you switch and the mission is in some kind of paused state?

**Review by hamishwillee — APPROVED** [review]
Thanks @sfuhrer ! Merging but might make post process changes based on your answer to https://github.com/PX4/PX4-user_guide/pull/3424/files#r1825185926

File: en/flight_modes_fw/mission.md (hunk 2 above)
Comment (sfuhrer) [reply en/flight_modes_fw/mission.md:79]: It prevents switching into Mission and will instead switch to Hold.

Comment (hamishwillee) [reply en/flight_modes_fw/mission.md:79]: Cool. Captured in https://github.com/PX4/PX4-user_guide/pull/3437

**Pull request merged**